docs/content/en/components/k8ssandra-operator/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "K8ssandra operator"
+linkTitle: "K8ssandra operator"
+weight: 1
+description: An introduction to K8ssandra operator, a v2 feature.
+---
+
+The K8ssandra operator provides multi-cluster and multi-region support for Apache Cassandra&reg; databases deployed in Kubernetes environments. K8ssandra operator also works in single-cluster and single-region Kubernetes deployments.
+
+You can install K8ssandra operator in:
+
+* A local Kubernetes 1.19+ environment, such as kind, K3D, or minikube
+* A supported cloud provider's Kubernetes environment:
+    * Amazon AKS
+    * Google Cloud GKE
+    * Digital Ocean
+    * Microsoft Azure
+
+## Architecture
+
+K8ssandra Operator consists of two primary components:
+
+* A control plane
+* A data plane
+
+ The control plane creates and manages objects that exist only in the API server. The control plane does not deploy or manage pods.
+
+{{% alert title="Note" color="success" %}}
+The control plane can be installed in only one cluster; that is, in the control plane cluster.
+{{% alert %}}
+
+The data plane can be installed on any number of clusters. The control plane cluster can also function as the data plane.
+
+The data plane deploys and manages pods. Moreover, the data plane may interact directly with the managed applications. For example, the operator may call the `management-api` to create keyspaces in Cassandra.
+
+In each cluster, the deployed and managed pods can include [Stargate]({{< relref "/components/stargate/" >}}) and [cass-operator]({{< relref "/components/cass-operator/" >}}).   
+
+Here's how the K8ssandra operator components fit together:
+
+![How the K8ssandra operator fit together](k8ssandra-operator-architecture.png)
+
+## Multi-cluster requirements
+
+There must exist routable pod IPs between Kubernetes clusters; however this requirement may be relaxed in the future.
+
+If you are running in a cloud provider, you can get routable IPs by installing the Kubernetes clusters in the same VPC.
```

Comment:
This is detail that we can save for later when we get into the different provider install processes, but this might not be strictly true across the board with each.

Reply:
Just leaving a note here so that we keep that in mind.